server/app/mutations/stacks/common.rb

```diff
@@ -14,6 +14,12 @@ def handle_service_outcome_errors(service_name, messages, type)
       end
     end
 
+    def handle_volume_outcome_errors(volume_name, errors)
+      errors.each do |key, atom|
+         add_service_error(volume_name, key, atom.symbolic, atom.message)
```

Comment:
This uses the  #1976 `add_service_error` which is not yet in git master, and adds a `services.$volume.$key` error, which doesn't make very much sense. See previous comment: https://github.com/kontena/kontena/pull/1849#discussion_r107435002

```ruby
add_error("volumes.#{volume_name}.#{key}", atom.symbolic, message)
```